exec/java-exec/src/test/java/org/apache/drill/exec/physical/resultSet/impl/TestResultSetLoaderOverflow.java

```diff
@@ -59,55 +61,64 @@
 
   @Test
   public void testVectorSizeLimit() {
-    TupleMetadata schema = new SchemaBuilder()
+    final TupleMetadata schema = new SchemaBuilder()
```

Comment:
Since Java supports effective final there is no need to excessive use of final keyword unless s you want to explicitly indicate that variable is final. I am not going to request the change during code review, I guess this mostly relates to the developers code writing style but since you are making lots of refactoring adding final keyword, just wanted to highlight that this is might be unnecessary.